packages/discord-verify/package.json

```diff
@@ -1,29 +1,58 @@
 {
 	"name": "discord-verify",
-	"version": "0.0.1",
+	"version": "1.0.0",
 	"author": "Ian Mitchell",
 	"description": "A library for verifying the authenticity of requests coming from the Discord Interactions API",
 	"license": "MIT",
 	"homepage": "https://github.com/ianmitchell/interaction-kit/tree/main/packages/discord-verify",
-	"exports": "./dist/src/verify.js",
-	"main": "./dist/src/verify.js",
-	"typings": "dist/src/verify.d.ts",
-	"type": "module",
+	"exports": {
+		".": {
+			"import": "./dist/web.mjs",
+			"require": "./dist/web.cjs",
+			"types": "./dist/web.d.ts"
+		},
+		"./web": {
+			"import": "./dist/web.mjs",
+			"require": "./dist/web.cjs",
+			"types": "./dist/web.d.ts"
+		},
+		"./node": {
+			"import": "./dist/node.mjs",
+			"require": "./dist/node.cjs",
+			"types": "./dist/node.d.ts"
+		}
+	},
+	"main": "./dist/web.mjs",
+	"types": "./dist/web.d.ts",
+	"typesVersions": {
```

Comment:
typesversions is here for legacy typescript support right?

Reply:
Yeah